ibis/legacy/udf/vectorized.py

```diff
@@ -350,6 +353,7 @@ def elementwise(input_type, output_type):
     return _udf_decorator(ElementWiseVectorizedUDF, input_type, output_type)
 
 
+@deprecated(as_of="9.0", instead="")
```

Comment:
Builtin reductions can still be accessed. Perhaps it is fine (and helpful?) to say `"use the `@ibis.udf.agg.builtin` decorator"` or something similar.